mkt/webapps/management/commands/fix_excluded_regions.py

```diff
@@ -1,34 +0,0 @@
-from optparse import make_option
-
-from django.core.management.base import BaseCommand
-
-from mkt.site.utils import chunked
-from mkt.webapps.models import Webapp
-from mkt.webapps.tasks import fix_excluded_regions
-
-
-HELP = """\
-    Usage:
-
-        python manage.py fix_excluded_regions [--app=<app_id>]
-
-    If no --app provided, all unrestricted apps are processed.
```

Comment:
This command was created a year ago to fix broken region exclusions in https://bugzilla.mozilla.org/show_bug.cgi?id=1055513 ; AFAIK it was only used once, we don't need it anymore.